pkgs/development/tools/build-managers/waf/setup-hook.sh

```diff
@@ -0,0 +1,62 @@
+wafConfigurePhase() {
+    runHook preConfigure
+
+    if ! [ -f ./waf ]; then
+        cp @waf@ waf
+    fi
```

Comment:
Out of curiosity, why did you elect to use `cp` instead of `ln`?

Reply:
I can't remember tbh. ln might have been a better idea but I think I had a package that was already doing this in configurePhase that used cp.